### ISSUE TITLE
DetailList - select all button, adding aria-describedby attribute even if no label is provided

### ISSUE THREAD

**Issue by kmrrakesh** [OfficeDev/office-ui-fabric-react#6638]
### Bug Report 
- __Package version(s)__: ">= 6.0.0 < 7" 
 - __Browser and OS versions__: Edge and Windows 10
 
#### Describe the issue:
DetailList - select all button, adding aria-describedby attribute even if no label is provided
 
#### Actual behavior:
DetailList - select all button, adding aria-describedby attribute even if no label is provided
 
#### Expected behavior:
DetailList - select all button, should not add "aria-describedby" attribute if no label is provided

![1b668f5b](https://user-images.githubusercontent.com/13743867/46736472-c5d53a80-ccb6-11e8-9029-56b2c2607d9e.PNG)
